doc/developer/topotests.rst

```diff
@@ -953,6 +953,32 @@ configuration files (per-daemon) using the following commands:
    end
    r1#
 
+You can also login to the node specified by nsenter using bash, etc.
+A pid file for each node will be created in the relevant test dir.
+You can run scripts inside the node, or use vtysh's <tab> or <?> feature.
+
+.. code:: shell
+
+  [unet shell]
+  # cd tests/topotests/srv6_locator
+  # ./test_srv6_locator.py --topology-only
+  unet> r1 sh segment-routing srv6 locator
```

Comment:
Given there is a `sh` (shell) CLI command (i.e., `unet> sh r1 ls -l`) I think it would be good to not abbreviate `show` as `sh` above.  It confused me (although I might be the only one confused by this :) ) Would you mind updated it to be `show` instead?

Reply:
Ah I see. This is confusing. I just update all omit.

Reply:
updated!